src/amplitude.js

```diff
@@ -1,314 +1,311 @@
 'use strict';
```

Comment:
I have a different view over this, actually our modules are transpiled into es5 or lower by babel,
So in browser our code would still be non es6 and hence browser need this
*use strict*
Declaration 
For operating in strict mode.

Reply:
hmm, that's a good point! Still, we don't need strict mode in our modules (any files in the `src` directory) because they're already in strict mode even after compilation with es6.

We can verify that by removing strict mode, and then doing any of the things that aren't allowed in es5's strict mode from this article https://flaviocopes.com/javascript-strict-mode/  and verifying that it still throws an error for the developer. My understanding of strict mode is that its audience is primarily developers because it throws errors that would otherwise be silent in ES5. So people who are working on p5.sound library will still get the same errors now that we're using ES6 modules. Meanwhile, people using the compiled library won't be modifying the library, so they don't need the library to be in strict mode.

We do still want to have `'use strict';` in in test files, since we haven't turned those into es6 modules yet.

Reply:
okay ! that's the nice  point to consider and i will be removing the 'strict mode'

Reply:
sucessfully removed the 'strict mode' declaration!